### PR TITLE
feat: cosmostation wallet integration

### PR DESCRIPTION
**Pull request by honeymaro**

### CODE REVIEW

File: src/components/Modal/ConnectWalletModal/index.tsx [hunk 1]
```diff
@@ -79,6 +82,21 @@ function ConnectWalletModal(props: ReactModal.Props) {
             ]
           : (p as WalletButtonProps),
       ),
+    ...(cosmostationWallet.isInstalled
```
Comment (jhlee-young) [inline src/components/Modal/ConnectWalletModal/index.tsx:85]: plz add its installations as well

File: src/hooks/useCosmostationWallet.ts [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        disconnect();
+      });
+    }
+  }, [cosmostationWallet, disconnect]);
+
+  useEffect(() => {
+    if (
+      // If the user is connected to the wallet, but read-only
+      xplaWallet.status === WalletStatus.WALLET_CONNECTED &&
+      !xplaWallet.supportFeatures.has("post")
```
Comment (jhlee-young) [inline src/hooks/useCosmostationWallet.ts:163]: wouldn't it be better to compare `xplaWallet.connection.type`?

File: src/hooks/useCosmostationWallet.ts.bak [hunk 3]
```diff
@@ -0,0 +1,209 @@
+import { useCallback, useEffect, useState } from "react";
```
Comment (jhlee-young) [inline src/hooks/useCosmostationWallet.ts.bak:1]: could you let me know what this file is for?

Comment (honeymaro) [reply src/hooks/useCosmostationWallet.ts.bak:1]: added this file accidentally

**Comment by honeymaro** [comment]
rebased

**Review by jhlee-young — APPROVED** [review]
LGTM

**Review by jhlee-young — APPROVED** [review]

**Pull request merged**